src/screens/ContactScreen/index.js

```diff
@@ -203,6 +227,29 @@ const ContactScreen = ({navigation}) => {
     }
   };
 
+  const handleInviteCommunityMember = async () => {
+    try {
+      const response = await inviteCommunityMember(
+        topicCommunityId.toString(),
+        selectedUsers?.map((user) => user?.user_id)
+      );
+      if (response.success) {
+        navigation.replace(NavigationConstants.CREATE_POST_SCREEN, {
+          isCreateCommunity: true,
+          topic: topicCommunityName
+        });
+      }
+    } catch (e) {
+      if (__DEV__) {
+        console.log('error handleInviteCommunityMember: ', e);
+      }
+    }
+  };
+
+  const onCommunityShare = () => {
+    ShareUtils.shareCommunity(topicCommunityName);
+  };
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The error handling in `handleInviteCommunityMember` function could be improved. Currently, it only logs the error when the app is in development mode. It would be better to provide some feedback to the user when an error occurs, such as displaying a message or alert.

```diff
  const handleInviteCommunityMember = async () => {
    try {
      const response = await inviteCommunityMember(
        topicCommunityId.toString(),
        selectedUsers?.map((user) => user?.user_id)
      );
      if (response.success) {
        navigation.replace(NavigationConstants.CREATE_POST_SCREEN, {
          isCreateCommunity: true,
          topic: topicCommunityName
        });
      }
    } catch (e) {
-     if (__DEV__) {
-       console.log('error handleInviteCommunityMember: ', e);
-     }
+     // Display an error message to the user
+     Alert.alert("Error", "Failed to invite community member.");
    }
  };
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->